src/components/Filters.js

```diff
@@ -171,6 +171,83 @@ function Filters() {
             </label>
           </ul>
         </div>
+        <div class="radio-stars">
+          <ul class="list">
+            <div class="list-heading">
+              <span>By Rating</span>
```

Comment:
Instead of writing 5 times rating `div` you can put `[1,2,3,4,5]` in an array and then map over it , your code will be reduced by only 1 single `Div`.